build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
                     "org.apache.taglibs:taglibs-standard-impl:1.2.5",
                     "org.jsoup:jsoup:1.9.2"
 
+    compile group: 'com.googlecode.owasp-java-html-sanitizer', name: 'owasp-java-html-sanitizer', version: 'r136'
```

Comment:
Let's format this similarly:
`"com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:r136"`
and put it on alphabetical order with the other compile-scope dependencies.

Also, I just checked the Maven repository and the latest version of this library is `r239`. Any reason for using `r136`?